mmv1/third_party/terraform/go.mod.erb

```diff
@@ -3,95 +3,101 @@ module github.com/hashicorp/terraform-provider-google
 go 1.19
 
 require (
-	cloud.google.com/go/bigtable v1.19.0
-	github.com/GoogleCloudPlatform/declarative-resource-client-library v1.52.0
+	cloud.google.com/go/bigtable v1.20.0
+	github.com/GoogleCloudPlatform/declarative-resource-client-library v1.54.0
```

Comment:
Was this PR intended to upgrade every available dependency?

This upgrades the DCL library and seems completely unrelated to DCL changes. Were the test failures from https://github.com/GoogleCloudPlatform/magic-modules/pull/9341#issuecomment-1780885768 addressed? 

I can only imagine potential side effects from upgrading all of these imports at once. @trodge

Reply:
I have reverted this PR.

Please open a new one that updates only the necessary dependencies.